packages/medusa/src/services/order.ts

```diff
@@ -1484,7 +1486,23 @@ class OrderService extends TransactionBaseService {
         .withTransaction(manager)
         .refundPayment(order.payments, refundAmount, reason, note)
 
-      const result = await this.retrieve(orderId)
+      let result = await this.retrieve(orderId, {
```

Comment:
todo: Use retrieveWithTotals instead which is more performant and will return the same result. We are trying to avoid additional legacy usage. 💪